src/Amazon.Extensions.CognitoAuthentication/CognitoAuthenticationClasses.cs

```diff
@@ -94,7 +99,11 @@ public class InitiateSrpAuthRequest
         /// <summary>
         /// The client metadata for the current authentication flow.
         /// </summary>
-        public IDictionary<string, string> ClientMetadata { get; set; }
+        public IDictionary<string, string> ClientMetadata { get; set; }
+        /// <summary>
+        /// The analytics metadata for collecting Amazon Pinpoint metrics.
+        /// </summary>
+        public AnalyticsMetadataType AnalyticsMetadata { get; set; }
```

Comment:
nit: add whitespace between the 2 properties

Reply:
Tried to follow the same structure as other properties where there is no whitespace. :)